Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:16-alpine
+ARG PNPM_VERSION=7.1.6
+RUN npm --location=global install pnpm@${PNPM_VERSION}
```

Comment:
We should use `corepack` for this.

Reply:
@antfu Thanks tony. I'll submit a change to use `corepack`